cmd/manager-hub/main.go

```diff
@@ -106,17 +106,19 @@ func main() {
 
 	registryAPI := registry.NewRegistry()
 	jobHelperAPI := utils.NewJobHelper(client)
+	buildHelper := build.NewHelper()
+	signHelper := sign.NewSignerHelper()
```

Comment:
That variable is not strictly needed as it's used only once.

Reply:
fixed